inference-engine/src/inference_engine/src/ie_network_reader.cpp

```diff
@@ -169,35 +176,33 @@ CNNNetwork details::ReadNetwork(const std::string& modelPath,
 #else
     std::string model_path = modelPath;
 #endif
+
     // Try to open model file
     std::ifstream modelStream(model_path, std::ios::binary);
     if (!modelStream.is_open())
         IE_THROW() << "Model file " << modelPath << " cannot be opened!";
 
     assertIfIRv7LikeModel(modelStream);
 
+    std::string bPath = binPath;
+    if (bPath.empty()) {
+        auto pathWoExt = modelPath;
+        auto pos = modelPath.rfind('.');
+        if (pos != std::string::npos)
+            pathWoExt = modelPath.substr(0, pos);
+
+        bPath = pathWoExt + ".bin";
+        if (!FileUtils::fileExist(bPath)) {
+            bPath.clear();
+        }
+    }
```

Comment:
looks like the logic above is IR specific, can this be a part of IE FE?

Reply:
moved to IR FE.